### PR TITLE
Run native image tests in CI

### PR DESCRIPTION
**Pull request by yschimke**
One blocking issue with gradle 7 and native image
https://github.com/palantir/gradle-graal/issues/421

### CODE REVIEW

**Comment by yschimke** [comment]
```
[ConsoleLauncher:96147]        image:   6,587.24 ms,  6.13 GB
[ConsoleLauncher:96147]        write:   1,695.44 ms,  6.13 GB
# Printing build artifacts to: build/graal/ConsoleLauncher.build_artifacts.txt
[ConsoleLauncher:96147]      [total]: 108,110.02 ms,  6.13 GB
native image available at build/graal/ConsoleLauncher (71 MB)

BUILD SUCCESSFUL in 2m 32s
45 actionable tasks: 36 executed, 9 up-to-date
yschimke-macbookpro2% ./native-image-tests/build/graal/ConsoleLauncher
Missing test class: java.lang.ClassNotFoundException: okhttp3.internal.io.FileSystemTest
...............................................................................
```

**Comment by yschimke** [comment]
Reported here https://github.com/oracle/graal/issues/3466

**Review by swankjesse — APPROVED** [review]

File: native-image-tests/build.gradle [hunk 1]
```diff
@@ -28,13 +28,7 @@ dependencies {
   implementation Dependencies.junit5Api
   implementation Dependencies.junit5JupiterParams
 
-  implementation("org.graalvm.nativeimage:svm:21.0.0") {
-    // https://youtrack.jetbrains.com/issue/KT-29513
-    exclude group: "org.graalvm.nativeimage"
```
Comment (swankjesse) [inline native-image-tests/build.gradle:33]: Yay removing excludes!

**Review by swankjesse — COMMENTED** [review]
LGTM

**Pull request merged**